azure-devops/azext_devops/dev/admin/banner.py

```diff
@@ -82,7 +82,10 @@ def banner_update(message=None, banner_type=None, id=None, expiration=None, orga
         raise ValueError('At least one of the following arguments need to be supplied: --message, --type, '
                          '--expiration.')
     if expiration is not None:
```

Comment:
something like this?
``` python
if expiration is not None and expiration != '':
    expiration_iso8601 = convert_date_string_to_iso8601(value=expiration, argument='expiration')
else:
    expiration_iso8601 = expiration
```

btw: is None and `` behave differently while calling API?

Reply:
Yes, to unset the date we explicitly have to pass empty string [i.e ''] and not None.
In case of none, expiration date will remain unchanged.

Reply:
For future reference.. More pythonic way of testing a string as none or empty.

```
if expiration: 
```